layer/runtime_layer.cc

```diff
@@ -393,6 +394,8 @@ SPL_RUNTIME_LAYER_FUNC(VkResult, CreateDevice,
     SPL_DISPATCH_DEVICE_FUNC(CreateQueryPool);
     SPL_DISPATCH_DEVICE_FUNC(DestroyQueryPool);
     SPL_DISPATCH_DEVICE_FUNC(GetQueryPoolResults);
+    SPL_DISPATCH_DEVICE_FUNC(FreeCommandBuffers);
```

Comment:
nit: can you keep this in the alphabetical order?

Reply:
Sure! I reordered the other ones as well to be in the alphabetical order.